AppKit/CPCompatibility.j

```diff
@@ -179,7 +182,10 @@ else if (USER_AGENT.indexOf("AppleWebKit/") != -1)
         PLATFORM_BUGS |= CPJavaScriptPasteRequiresEditableTarget;
         // https://bugs.webkit.org/show_bug.cgi?id=39689
         PLATFORM_BUGS |= CPJavaScriptPasteCantRefocus;
+        PLATFORM_FEATURES[CPTextSizingAlwaysNeedsSetFontFeature] = YES;
```

Comment:
If there is a way to detect this directly, it's always preferable to a simple engine/browser detection.